lib/screens/creator_hub/creator_hub_screen.dart

```diff
@@ -159,83 +142,176 @@ class _CreatorHubScreenState extends State<CreatorHubScreen> {
+                  ),
+                ),
+              ),
+              SizedBox(
+                width: 10.w,
+              ),
+              Expanded(
                 child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
                   children: [
-                    Stack(
-                      alignment: AlignmentDirectional.bottomStart,
-                      children: [
-                        Row(
-                          children: [
-                            SizedBox(
-                              width: 120.w,
-                              height: 23.h,
-                              child: Text(
-                                title,
-                                maxLines: 1,
-                                style: EaselAppTheme.titleStyle.copyWith(fontSize: 15.sp),
-                              ),
-                            ),
-                            SizedBox(
-                              width: 20.w,
-                            ),
-                            GestureDetector(
-                              behavior: HitTestBehavior.translucent,
-                              onTap: () {
-                                viewModel.draftCollapse = !viewModel.draftCollapse;
-                              },
-                              child: SizedBox(height: 20.h, width: 20.w, child: Icon(viewModel.draftCollapse ? Icons.add : Icons.remove)),
-                            )
-                          ],
-                        ),
-                        Wrap(
-                          children: [
-                            Container(
-                              width: 120.w,
-                              height: 10.h,
-                              decoration: const BoxDecoration(border: Border(bottom: BorderSide(color: EaselAppTheme.kBlack, width: 2))),
-                            ),
-                            CustomPaint(size: Size(10.w, 10.h), painter: DiagonalLinePainter()),
-                          ],
-                        ),
-                      ],
+                    Text(
+                      "NFT name",
+                      style: titleStyle,
+                      maxLines: 1,
+                      overflow: TextOverflow.ellipsis,
                     ),
                     SizedBox(
-                      height: 10.h,
+                      height: 11.h,
+                    ),
+                    Text(
+                      "draft".tr(),
+                      style: titleStyle.copyWith(color: EaselAppTheme.kLightRed, fontSize: 13.sp),
                     ),
-                    viewModel.draftCollapse ? const SizedBox() : Container(),
                   ],
                 ),
               ),
-            ),
+              SizedBox(
+                width: 10.w,
+              ),
+              InkWell(
+                  onTap: () {
+                    showModalBottomSheet(
```

Comment:
showModalBottomSheet should be in the class . Check how i am showing receipt dialog in wallet.